src/functions.js

```diff
@@ -297,15 +297,24 @@ defineFunction([
 
 // No limits, symbols
 defineFunction([
-    "\\int", "\\iint", "\\iiint", "\\oint",
+    "\\int", "\\iint", "\\iiint", "\\oint", "∫", "\u222c", "\u222d", "\u222e",
```

Comment:
Is there a reason that `∫` gets embedded in Unicode and the others are `\u` escape sequences?  Something about the character being available in a default font?

Reply:
Your guess is on the money. I tend to code in the Consolas font, in which ∫ is available, but not the other characters in this PR. If you would like to see some of any of them changed, let me know.

Reply:
I feel like it'd be more consistent to put them all as `\u` escape sequences, but I don't feel strongly about it.

Reply:
Done.